### ISSUE TITLE
Error FaceEmbedDistance

### ISSUE THREAD

**Issue by 2PeteShakur** [cubiq/ComfyUI_FaceAnalysis#6]
Error occurred when executing FaceEmbedDistance:

No image matches the filter criteria.

File "E: \ComfyUI_portable\ComfyUI\execution.py", line 152, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
File "E: \ComfyUI_portable\ComfyUI\execution.py", line 82, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj. FUNCTION, allow_interrupt=True)
File "E: \ComfyUI_portable\ComfyUI\execution.py", line 75, in map_node_over_list
results.append(getattr(obj, func) ( ** slice_dict(input_data_all, i)))
File "E: \ComfyUI_portable\ComfyUI\custom_nodes \ComfyUI_FaceAnalysis\faceanalysis.py", line 239, in analize
raise Exception('No image matches the filter criteria. ')

Got the above error when attempting usage

**Comment by andupotorac** [comment]
Same here, but when using two images which are not the same person.

**Comment by electblake** [comment]
I fixed but been editing code myself. I doubt I'll take the time to make sure the change is actually safe to share.

tldr; instead of throwing exception I generate a black image, no errors and black images generate in my workflow machines anyway (with VRAM limits)

glhg
